music_assistant/server/providers/opensubsonic/sonic_provider.py

```diff
@@ -792,6 +792,11 @@ async def get_stream_details(self, item_id: str) -> StreamDetails:
         if mime_type.endswith("mpeg"):
             mime_type = sonic_song.suffix
 
+        logging.getLogger("libopensonic").debug(
```

Comment:
I think this should probably use `self.logger`.

Logging generally does not use f-strings. This is a small efficiency saving as you can defer the string concatenation (and associated memory allocations), not even doing it at all if the debug log is off.

Reply:
Yep, looks great to me.